adding_species_manuscript.tex

```diff
@@ -658,9 +658,8 @@ \subsection*{Limitations of simulated genomes}
 % by the final statement in this paragraph. We can uncomment, if needed.
 %(albeit the genealogy of any set of sampled individuals includes many ancestors).
 Thus, demographic models inferred from larger sets of samples from diverse ancestry backgrounds
-may provide a more comprehensive depiction of genetic variation within a species,
-as long as a sufficiently realistic demographic model can be fit
-(in particular, models that account for the structure of populations within a species).
+may potentially provide a more comprehensive depiction of genetic variation within a species.
+This is true if sufficiently realistic demographic model can be fit---models that account for the structure of populations within a species.
```

Comment:
Are there other potentially important aspects of realism here? If so, perhaps "-- for instance, models that account for the structure of populations within a species."